modules/virt-cloning-a-datavolume.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc
+
+// `blockstorage` conditionals are used (declared in the "*-block" assembly) to separate content
+
+[id="virt-cloning-a-datavolume_{context}"]
+= Cloning a DataVolume
+
+== Prerequisites
```

Comment:
This should be ".Prerequisites" since it's in a module now.

Reply:
Gotcha. Makes sense.